app/MMB/src/components/exploreCity/ExploreText.js

```diff
@@ -5,25 +5,17 @@ import { View } from 'react-native';
 import { type TranslationType } from '@kiwicom/mobile-localization';
 import { StyleSheet, Text, Color } from '@kiwicom/mobile-shared';
 
-import ExploreButton from './ExploreButton';
-
 type Props = {|
   +title: TranslationType,
-  +text: TranslationType,
-  +city: ?string,
+  +text: TranslationType | $ReadOnlyArray<TranslationType>,
 |};
 
 const ExploreText = (props: Props) => {
   return (
-    <React.Fragment>
-      <View style={styles.padding}>
-        <Text style={styles.title}>{props.title}</Text>
-        <Text style={styles.subText}>{props.text}</Text>
-      </View>
-      <View style={[styles.padding, styles.buttonContainer]}>
-        <ExploreButton cityName={props.city} />
-      </View>
-    </React.Fragment>
+    <View style={styles.padding}>
+      <Text style={styles.title}>{props.title}</Text>
+      <Text style={styles.subText}>{props.text}</Text>
```

Comment:
I don't know if that suggestion makes sense, but you could have added it here, instead of manually?

Reply:
Sorry, I could have added what?